package.json

```diff
@@ -35,8 +35,8 @@
   "devDependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/browserslist-config": "1.2.0",
-    "@openedx/frontend-build": "14.0.3",
-    "@openedx/paragon": "22.3.1",
```

Comment:
Why do we want to lower the version of Paragon? Is it just the PR is old?

Reply:
it was synced with old alpha branch, let me rebase it